<?xml version="1.0"?>

<launch>
    <arg name="ur_hardware_interface" default="/ur_hardware_interface"/>
    <arg name="joint" type="str" default="gripper_motor_joint"/>
    <arg name="max_position_voltage" default="3.0"/>
    <arg name="state_publish_rate" default="50.0"/>
    <arg name="action_monitor_rate" default="20.0"/>

    <node name="onrobot_gripper" pkg="onrobot_control" type="onrobot_gripper_node.py" output="screen" respawn="true">
        <param name="ur_hardware_interface" type="str" default="$(arg ur_hardware_interface)"/>
        <param name="joint" type="str" default="$(arg joint)"/>
        <param name="max_position_voltage" type="float" default="$(arg max_position_voltage)"/>
        <param name="state_publish_rate" type="float" default="$(arg state_publish_rate)"/>
        <param name="action_monitor_rate" type="float" default="$(arg action_monitor_rate)"/>
    </node>
</launch>